lib/rdoc/generator/template/aliki/_head.rhtml

```diff
@@ -4,88 +4,155 @@
-    <meta name="twitter:description" content="Documentation for <%= h klass.full_name %> <%= h klass.type %> - API reference and code examples">
+    <meta
+      name="twitter:description"
+      content="Documentation for <%= h klass.full_name %> <%= h klass.type %> - API reference and code examples"
+    >
   <%- else %>
     <meta name="twitter:description" content="<%= h excerpt(klass.comment) %>">
   <%- end %>
 <%- elsif defined?(file) %>
   <%- if file.comment.empty? %>
-    <meta name="twitter:description" content="<%= h file.page_name %> - <%= h @title %> documentation">
+    <meta
+      name="twitter:description"
+      content="<%= h file.page_name %> - <%= h @title %> documentation"
+    >
   <%- else %>
     <meta name="twitter:description" content="<%= h excerpt(file.comment) %>">
   <%- end %>
 <%- else %>
-  <meta name="twitter:description" content="API documentation for <%= h @title %> - Browse classes, modules, and methods">
+  <meta
+    name="twitter:description"
+    content="API documentation for <%= h @title %> - Browse classes, modules, and methods"
+  >
 <%- end %>
 
 <script type="text/javascript">
   var rdoc_rel_prefix = "<%= h asset_rel_prefix %>/";
   var index_rel_prefix = "<%= h rel_prefix %>/";
 </script>
 
-<script src="<%= h asset_rel_prefix %>/js/theme-toggle.js"></script>
-<script src="<%= h asset_rel_prefix %>/js/navigation.js" defer></script>
-<script src="<%= h asset_rel_prefix %>/js/search.js" defer></script>
-<script src="<%= h asset_rel_prefix %>/js/search_index.js" defer></script>
-<script src="<%= h asset_rel_prefix %>/js/searcher.js" defer></script>
-<script src="<%= h asset_rel_prefix %>/js/c_highlighter.js" defer></script>
-<script src="<%= h asset_rel_prefix %>/js/aliki.js" defer></script>
+<script
+  src="<%= h asset_rel_prefix %>/js/theme-toggle.js?v=<%= h RDoc::VERSION %>"
+></script>
+
+<script
+  src="<%= h asset_rel_prefix %>/js/navigation.js?v=<%= h RDoc::VERSION %>"
+  defer
+></script>
+
+<script
+  src="<%= h asset_rel_prefix %>/js/search.js?v=<%= h RDoc::VERSION %>"
+  defer
+></script>
+
+<script
+  src="<%= h asset_rel_prefix %>/js/search_index.js?v=<%= h RDoc::VERSION %>"
+  defer
+></script>
+
+<script
+  src="<%= h asset_rel_prefix %>/js/searcher.js?v=<%= h RDoc::VERSION %>"
+  defer
+></script>
+
+<script
+  src="<%= h asset_rel_prefix %>/js/c_highlighter.js?v=<%= h RDoc::VERSION %>"
+  defer
+></script>
+
+<script
+  src="<%= h asset_rel_prefix %>/js/aliki.js?v=<%= h RDoc::VERSION %>"
+  defer
+></script>
+
+<link
+  href="<%= h asset_rel_prefix %>/css/rdoc.css?v=<%= h RDoc::VERSION %>"
+  rel="stylesheet"
+>
 
-<link href="<%= h asset_rel_prefix %>/css/rdoc.css" rel="stylesheet">
 <%- @options.template_stylesheets.each do |stylesheet| %>
-<link href="<%= h asset_rel_prefix %>/<%= File.basename stylesheet %>" rel="stylesheet">
+  <link
+    href="<%= h asset_rel_prefix %>/<%= h File.basename stylesheet %>?v=<%= h RDoc::VERSION %>"
+    rel="stylesheet"
+  >
```

Comment:
These styling change is suggested by Herb. I personally feel fine with them but am open to apply custom rules later if anyone has any strong opinion.